wilds/datasets/gwhd_dataset.py

```diff
@@ -0,0 +1,143 @@
+import numpy as np
+import pandas as pd
+import torch
+from pathlib import Path
+from PIL import Image
+from wilds.datasets.wilds_dataset import WILDSDataset
+from wilds.common.grouper import CombinatorialGrouper
+from wilds.common.metrics.all_metrics import DetectionAccuracy
+
+
+def _collate_fn(batch):
+    """
+    Stack x (batch[0]) and metadata (batch[2]), but not y.
+    """
+    batch = list(zip(*batch))
+    batch[0] = torch.stack(batch[0])
+    batch[2] = torch.stack(batch[2])
+    return tuple(batch)
+
+class GWHDDataset(WILDSDataset):
+    """
+    The GWHD-wilds wheat head localization dataset.
+    This is a modified version of the original Global Wheat Head Dataset.
+    This dataset is not part of the official WILDS benchmark.
+    We provide it for convenience and to reproduce observations discussed in the WILDS paper.
+    Supported `split_scheme`:
+        'official' for WILDS related tasks.
+        To reproduce the baseline, several splits are needed:
+        - to train a model on train domains and test against a all test split: 'train_in-dist'
+        - "benchmark_biased" ; "benchmark_in-dist"
+    Input (x):
+        1024x1024 RGB images of wheat field canopy between flowering and ripening.
+    Output (y):
+        y is a nx4-dimensional vector where each line represents a box coordinate (x_min,y_min,x_max,y_max)
+    Metadata:
+        Each image is annotated with the ID of the domain it came from (integer from 0 to 10).
+    Website:
+        http://www.global-wheat.com/
+    Original publication:
+        @article{david_global_2020,
+            title = {Global {Wheat} {Head} {Detection} ({GWHD}) {Dataset}: {A} {Large} and {Diverse} {Dataset} of {High}-{Resolution} {RGB}-{Labelled} {Images} to {Develop} and {Benchmark} {Wheat} {Head} {Detection} {Methods}},
+            volume = {2020},
+            url = {https://doi.org/10.34133/2020/3521852},
+            doi = {10.34133/2020/3521852},
+            journal = {Plant Phenomics},
+            author = {David, Etienne and Madec, Simon and Sadeghi-Tehran, Pouria and Aasen, Helge and Zheng, Bangyou and Liu, Shouyang and Kirchgessner, Norbert and Ishikawa, Goro and Nagasawa, Koichi and Badhon, Minhajul A. and Pozniak, Curtis and de Solan, Benoit and Hund, Andreas and Chapman, Scott C. and Baret, Frédéric and Stavness, Ian and Guo, Wei},
+            month = aug,
+            year = {2020},
+            note = {Publisher: AAAS},
+            pages = {3521852},
+        }
+    License:
+        This dataset is distributed under the MIT license.
+        https://github.com/snap-stanford/ogb/blob/master/LICENSE
+    """
+
+    _dataset_name = 'gwhd'
+    _versions_dict = {
+        '2.0': {

```

Comment:
Could we perhaps set the version as `0.9`? And then when we release the private test data after the competition ends, we can bump it to `1.0`?

Reply:
Also, could we change the name of the RELEASE file in the bundle to reflect the version please? Thanks!